### DISCUSSION TITLE
How can the settings of this plugin be used in vim?

### DISCUSSION THREAD

**Discussion by nimser** [PatrickF1/fzf.fish#269] [Q&A] [answered]
I have neovim with fzf and wondered what would be the steps to port the current fish conf to work when fzf in used within vim.

**Comment by PatrickF1** [comment, marked as answer]
I'm sorry I don't use Vim. However, I can say confidently this plugin only works within the context of fish shell, so you'd need to spawn a fish shell within Vim to make it work.
This might be closer to what you're looking for. Good luck! https://github.com/junegunn/fzf.vim
